packages/contracts/docs/core/01-how-it-works/01-core/02-permissions/01-conditions.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Conditions
+---
+
+## Permission Conditions
```

Comment:
Maybe it's more helpful to talk about "conditional permissions" in general, instead of permission conditions.

Reply:
I prefer to leave it as it is to have the connection with the `PermissionCondition` contract